### PR TITLE
[WIP] feat: comment  toToast Messages  is displayed NOT scrolled anymore.

### LINKED ISSUE #3000: 💬 [Question] which possibilities would you suggest to make scrolling unnecessary?
> [ogerly](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-02-05T11:03:21Z" title="Wednesday, February 5th 2020, 12:03:21 pm +01:00">Feb 5, 2020</time>_

---

## :speech_balloon: Question
if a user gets a lot of comments for a post then the following problem occurs. 
if you want to answer several users then it scrolls to the editor for each name. 
which possibilities would you suggest to make scrolling unnecessary? 
i see scrolling as an optical signal. we would have to find another signal which tells the user that the username is written in the answer field. 
maybe you should check if the username you want to include is the last one in the editor. 
or each reply button can only be triggered once as an event.

### PR DESCRIPTION
**Pull request by Tirokk**
> [ogerly](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-02-22T15:25:36Z" title="Saturday, February 22nd 2020, 4:25:36 pm +01:00">Feb 22, 2020</time>_
_Closed <time datetime="2020-03-11T10:42:28Z" title="Wednesday, March 11th 2020, 11:42:28 am +01:00">Mar 11, 2020</time>_
---

## 🍰 Pullrequest
when responding to a comment of a user a toast is displayed NOT scrolled anymore.

ezgif com-video-to-gif

### Issues
- fixes #3000

### CODE REVIEW

**Comment by Tirokk** [comment]
> [cypress[bot]](https://github.com/apps/cypress) **Authored by [cypress[bot]](https://github.com/apps/cypress)**
_<time datetime="2020-02-22T16:01:52Z" title="Saturday, February 22nd 2020, 5:01:52 pm +01:00">Feb 22, 2020</time>_

---



<p align="center">
  <br />
  
</p>

---

### Test summary
<a href="https://dashboard.cypress.io/projects/qa7fe2/runs/5532/specs"></a> <b title="Passed">66</b>  &bull; <a href="https://dashboard.cypress.io/projects/qa7fe2/runs/5532/failures"></a> <b title="Failed">0</b>  &bull;  <b title="Pending">0</b> &bull;  <b title="Skipped">0</b>

---

### Run details
| | |
| --- | --- |
| **Project** | Human-Connection |
| **Status** | Passed |
| **Commit** | ece9d3b8b6 |
| **Started** | Feb 22, 2020 8:24 PM |
| **Ended** | Feb 22, 2020 8:45 PM |
| **Duration** | 21:29 <a href="https://on.cypress.io/parallelization" title="Enable parallelization to reduce run duration">💡</a> |
| **OS** | Linux Ubuntu Linux - 16.04 |
| **Browser** | Firefox 68  |

[View run in Cypress Dashboard ➡️](https://dashboard.cypress.io/projects/qa7fe2/runs/5532/specs)



---

This comment has been generated by **cypress-bot** as a result of this project's GitHub integration settings. You can manage this integration in [this project's settings in the Cypress Dashboard](https://dashboard.cypress.io/projects/qa7fe2/settings)

**Comment by Tirokk** [comment]
> [cypress[bot]](https://github.com/apps/cypress) **Authored by [cypress[bot]](https://github.com/apps/cypress)**
_<time datetime="2020-02-22T16:04:17Z" title="Saturday, February 22nd 2020, 5:04:17 pm +01:00">Feb 22, 2020</time>_

---



<p align="center">
  <br />
  
</p>

---

### Test summary
<a href="https://dashboard.cypress.io/projects/qa7fe2/runs/5531/specs"></a> <b title="Passed">66</b>  &bull; <a href="https://dashboard.cypress.io/projects/qa7fe2/runs/5531/failures"></a> <b title="Failed">0</b>  &bull;  <b title="Pending">0</b> &bull;  <b title="Skipped">0</b>

---

### Run details
| | |
| --- | --- |
| **Project** | Human-Connection |
| **Status** | Passed |
| **Commit** | 0741218a04 <span style="cursor:help;" title="Your CI provider generated this commit by merging this branch into the target branch.">ℹ️</span> |
| **Started** | Feb 22, 2020 8:24 PM |
| **Ended** | Feb 22, 2020 8:47 PM |
| **Duration** | 22:54 <a href="https://on.cypress.io/parallelization" title="Enable parallelization to reduce run duration">💡</a> |
| **OS** | Linux Ubuntu Linux - 16.04 |
| **Browser** | Firefox 68  |

[View run in Cypress Dashboard ➡️](https://dashboard.cypress.io/projects/qa7fe2/runs/5531/specs)



---

This comment has been generated by **cypress-bot** as a result of this project's GitHub integration settings. You can manage this integration in [this project's settings in the Cypress Dashboard](https://dashboard.cypress.io/projects/qa7fe2/settings)

File: webapp/components/CommentCard/CommentCard.vue [hunk 1]
```diff
@@ -149,8 +148,9 @@ export default {
       return `#${this.anchor}` === anchor
     },
     reply() {
-      const message = { slug: this.comment.author.slug, id: this.comment.author.id }
+      const message = { slug: this.comment.author.slug + ' ', id: this.comment.author.id }
       this.$emit('reply', message)
+      this.$toast.success(this.$t(`post.comment.addReply`))
```
Comment (Tirokk) [inline webapp/components/CommentCard/CommentCard.vue:153]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-23T14:44:25Z" title="Sunday, February 23rd 2020, 3:44:25 pm +01:00">Feb 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -149,8 +148,9 @@ export default {
       return `#${this.anchor}` === anchor
     },
     reply() {
-      const message = { slug: this.comment.author.slug, id: this.comment.author.id }
+      const message = { slug: this.comment.author.slug + ' ', id: this.comment.author.id }
       this.$emit('reply', message)
+      this.$toast.success(this.$t(`post.comment.addReply`))
```


I really dislike this solution. You get a success message even before a comment has been saved.

**Comment by Tirokk** [comment]
> [ogerly](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-03-11T10:42:28Z" title="Wednesday, March 11th 2020, 11:42:28 am +01:00">Mar 11, 2020</time>_

---


this will not be implemented now

**Review by Mogge — CHANGES_REQUESTED** [review]
I would prefer a human solution over a technical solution. How about telling the user to type in the name? See my comment [here](https://github.com/Human-Connection/Human-Connection/issues/3000#issuecomment-590075771). Is it OK to close this PR unmerged?

**Pull request closed without merging**